src/js/core/animation_frame.js

```diff
@@ -1,7 +1,6 @@
 import { Signal } from "./signal";
 
-// @ts-ignore
-import BackgroundAnimationFrameEmitterWorker from "../webworkers/background_animation_frame_emittter.worker";
+import BackgroundAnimationFrameEmitterWorker from "worker-loader!../webworkers/background_animation_frame_emittter.worker";
```

Comment:
I think you need to pass the `fallback` and `inline` options here then, not?

Reply:
Sorry about that, for some reason I read their docs as inline being the default.